e2e/components/navigation/navigation.readonly.spec.js

```diff
@@ -97,4 +97,25 @@ describe('when logged in', () => {
       });
     });
   });
+  describe('as an Admin', () => {
```

Comment:
```suggestion
  describe('as an ADMIN user', () => {
```

or

```suggestion
  describe('as ROLE_ADMIN', () => {
```

Here the problem is that the word "Admin" is not specific to our ROLE_ADMIN, as it generally just means "an administrative user", which in our system could be any logged in user